test/e2e/memcached_test.go

```diff
@@ -217,17 +217,20 @@ func TestMemcached(t *testing.T) {
 		t.Fatalf("could not rename test/e2e/memcached_test.go.tmpl: %v\nCommand Output:\n%v", err, string(cmdOut))
 	}
 
+	// link local sdk to vendor if not in travis
+	if repo == "" {
+		for _, dir := range []string{"pkg", "internal"} {
+			repoDir := filepath.Join("github.com/operator-framework/operator-sdk", dir)
+			vendorDir := filepath.Join("vendor", repoDir)
+			os.RemoveAll(vendorDir)
+			os.Symlink(filepath.Join(gopath, "src", repoDir), vendorDir)
+		}
+	}
```

Comment:
Why was this moved before the `dep ensure`? `dep ensure` will just remove these symlinks when it runs. This should be done immediately after the `dep ensure`